code/bit_manipulation/src/bit_division/bitdivision.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+Problem Statement : 
+Divide two numbers using bitwise operators
+(i.e without using arithmetic operators)
+*/
+
+#include <iostream>
+using namespace std;
```

Comment:
Remove this.

Reply:
Do you mean, removal of "using namespace std" ??